packages/mobile/src/home/CashInBottomSheet.tsx

```diff
@@ -54,15 +124,35 @@ function CashInBottomSheet() {
         >
           <Times />
         </Touchable>
-        <Text style={styles.title}>{t('cashInBottomSheet.title')}</Text>
-        <Text style={styles.subtitle}>{t('cashInBottomSheet.subtitle')}</Text>
-        <Button
-          text={t('cashInBottomSheet.addFunds')}
-          size={BtnSizes.FULL}
-          onPress={goToAddFunds}
-          style={styles.addFundBtn}
-          testID={'cashInBtn'}
-        />
+        {rampAvailable && rampCashInButtonExpEnabled ? (
```

Comment:
just a thought! don't forget to ramp the experiment to regions where ramp is available

Reply:
Isn't it already limited by the check in the useAsync hook on `.unavailable` and `.restricted` ?

Reply:
It is already limited but if we dont limit the experiment to ramp only users, we will get diluted results from firebase.

Reply:
Ah I think there is a better way to do that with activation events. I'll add an event specifically for ramp availablity so that we can use it in the experiment!
<img width="712" alt="Screen Shot 2022-01-11 at 1 13 12 PM" src="https://user-images.githubusercontent.com/8432644/149022421-aad1d488-7f33-4574-a380-3b855f691389.png">

Reply:
https://github.com/valora-inc/wallet/pull/1736/files#diff-9536823e710c7215fefeaaa2da0274e5220c8e7629ff1aa6061442b5ba981fb4R73-R77

Reply:
very cool, nice find @jh2oman

Reply:
Ooo interesting! nice find!